zipline/gens/brokers/ib_broker.py

```diff
@@ -48,7 +48,17 @@
                                    'market_value', 'average_cost',
                                    'unrealized_pnl', 'realized_pnl',
                                    'account_name'])
-
+ExceptionSymbols = {}
+ExceptionSymbols['VIX']=['CBOE','IND']
+ExceptionSymbols['GLD']=['ARCA','STK']
+ExceptionSymbols['GDX']=['ARCA','STK']
+DefaultRouter = ['SMART','STK']
+
+def check_symbol(symbol)
```

Comment:
In general: if you add a function, please include a testcase for it.
This function is broken (missing `:` from the end of the definition).
Luckily won't be needed if we go with the defaultdict approach.

Reply:
thanks, sloppy indeed